shared/cert.go

```diff
@@ -245,13 +246,17 @@ func TestingAltKeyPair() *CertInfo {
  * Generate a list of names for which the certificate will be valid.
  * This will include the hostname and ip address.
```

Comment:
Worth updating the docstring saying that `name` will take precedence over the hostname?